boa3/model/builtin/native/neo_contract_methods/unvote.py

```diff
@@ -0,0 +1,35 @@
+from typing import Dict, List, Tuple
+
+from boa3.model.builtin.interop.nativecontract import NeoContractMethod
+from boa3.model.variable import Variable
+from boa3.neo.vm.opcode.Opcode import Opcode
+
+
+class UnVoteMethod(NeoContractMethod):
+
+    def __init__(self):
+        from boa3.model.type.type import Type
+        from boa3.model.type.collection.sequence.uint160type import UInt160Type
+
+        identifier = 'un_vote'
+        native_identifier = ''  # un_vote is not neo native
+        args: Dict[str, Variable] = {
+            'account': Variable(UInt160Type.build()),
+        }
+
+        super().__init__(identifier, native_identifier, args, return_type=Type.bool)
+
+    @property
+    def opcode(self) -> List[Tuple[Opcode, bytes]]:
+        from boa3.model.builtin.native.nativecontract import NativeContract
+        return (
+            [
+                (Opcode.UNPACK, b''),
+                (Opcode.DROP, b''),
+                (Opcode.PUSHNULL, b''),
+                (Opcode.SWAP, b''),
+                (Opcode.PUSH2, b''),
+                (Opcode.PACK, b''),
+            ] +
+            NativeContract.NEO.class_methods['vote'].opcode
+        )
```

Comment:
Given `native_identifier = 'vote'` as you first implemented, does the following produce a different result?
I don't think the import is justified if the output is the same
```suggestion
        return [
            (Opcode.UNPACK, b''),
            (Opcode.INC, b''),
            (Opcode.PUSHNULL, b''),
            (Opcode.SWAP, b''),
            (Opcode.PACK, b''),
        ] + super().opcode()
```

Reply:
I see, I'll use super, but I'll still use the other Opcodes, INC is more expensive than DROP + PUSH2.